lib/simple_saml/assertion.ex

```diff
@@ -187,4 +213,38 @@ defmodule SimpleSaml.Assertion do
         {:error, {:failed_to_parse_timestamp, iso8601_datetime}}
     end
   end
+
+  defp extract_attributes(node) do
+    with {:ok, attributes_node} <- XmlNode.first_child(node, "*:AttributeStatement") do
+      XmlNode.children(attributes_node, "*:Attribute")
```

Comment:
Perhaps ensuring extraction works for non-prefixed tags with:

```suggestion
    with {:ok, attributes_node} <- XmlNode.first_child(node, ~r/.*:?AttributeStatement$/) do
      XmlNode.children(attributes_node, ~r/.*:?Attribute$/")
```

would make sense here?

Reply:
@zoldar good shout, I've added that!